packages/docs/content/1_getting-started/1_using-sass-and-css.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: Using Sass and CSS
```

Comment:
I noticed that in the nav menu the case of "CSS" isn't maintained
<img width="261" alt="Screen Shot 2022-04-22 at 3 13 21 PM" src="https://user-images.githubusercontent.com/7595652/164802966-2134d320-d716-4f8f-aa04-e4b40b9adfd1.png">

Reply:
Fixed. I think that was because these nav labels are using the file name and it wasn't capitalized there